storage/postgres/storage.go

```diff
@@ -290,9 +284,10 @@ func (ps *PostgresStorage) InTransaction(ctx context.Context, f func(ctx context
 	}()
 
 	transactionalStorage := &PostgresStorage{
-		pgDB:   tx,
-		db:     ps.db,
-		scheme: ps.scheme,
+		pgDB:          tx,
+		db:            ps.db,
+		scheme:        ps.scheme,
+		encryptionKey: ps.encryptionKey,
```

Comment:
If we're not going to use this `encryptionKey` we should remove it

Reply:
Opening a transaction, and calling `txStorage.Security().Fetcher()` inside a transaction currently fails, because the encryption key is not set.